.gitignore

```diff
@@ -102,3 +102,6 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# vscode
+.vscode/
```

Comment:
As a note, standard practice is to add this files to your own personal `~/.git/info/exclude`. I don't particularly mind, but you will get pushback in other OSS projects.

Reply:
I see. Thanks for the advice!